example/lib/presentation/state/custom_clusterization_map_strategy.dart

```diff
@@ -0,0 +1,214 @@
+import 'package:fluster/fluster.dart';
+import 'package:flutter/material.dart';
+import 'package:yandex_mapkit_example/assets/res/assets.dart';
+import 'package:yandex_mapkit_example/presentation/state/map_strategy.dart';
+import 'package:yandex_mapkit_lite/yandex_mapkit_lite.dart';
+
+class CustomClusterizationMapStrategyDelegate extends MapStrategyDelegate {
+  @override
+  Widget buildActions(BuildContext context) => const SizedBox();
+
+  /// There is only one placemark on the map.
+  ///
+  /// The placemark is a circle with a red number inside, which is incremented
+  /// every time the user taps on the placemark.
+  ///
+  /// There is also a action button for incrementing.
+  @override
+  Set<MapObject> get mapObjects {
+    return _clusteredObjects.map((cluster) {
+      if (cluster.isCluster ?? false) {
+        return PlacemarkMapObject(
+          point: cluster.location,
+          icon: PlacemarkIcon.single(
+            PlacemarkIconStyle(
+              image: BitmapDescriptor.fromAssetImage(Assets.cluster),
+            ),
+          ),
+          consumeTapEvents: true,
+          text: PlacemarkText(
+            text: cluster.pointsSize.toString(),
+            style: const PlacemarkTextStyle(
+              color: Colors.black,
+              size: 10,
+            ),
+          ),
+          opacity: 1,
+          mapId: MapObjectId(cluster.clusterId!.toString()),
+        );
+      }
+
+      final id = int.parse(cluster.markerId!);
```

Comment:
not a safe operation. Use `tryParse` instead

Reply:
done